lib/jsdom/browser/default-stylesheet.js

```diff
@@ -30,8 +30,6 @@ module.exports = `
  *
  */
 
-@namespace "http://www.w3.org/1999/xhtml";
```

Comment:
It did, see https://gist.github.com/flaviut/9485daa073065e5944d769fd6900138f#file-errors-txt-L1

I've restored it though. It seems like a bug that could be fixed in another way.